PacketDefinitions420/PacketHandlerManager.cs

```diff
@@ -264,39 +267,36 @@ public bool HandlePacket(Peer peer, ENet.Packet packet, Channel channelId)
             }
             return HandlePacket(peer, data, channelId);
         }
+
         private bool HandleHandshake(Peer peer, byte[] data)
         {
             var request = PacketReader.ReadKeyCheckRequest(data);
             // TODO: keys for every player
             ulong playerID = (ulong)_blowfish.Decrypt(request.CheckId);
 
-            if (request.PlayerID != playerID)
+            if (request.PlayerID != playerID || _peers.ContainsKey(request.PlayerID))
             {
-                // wrong blowfish key
```

Comment:
You shouldn't remove these descriptive comments. Replace the comment with ` // Wrong blowfish key or player already connected`.